{{cookiecutter.project_slug}}/tests/test_{{cookiecutter.project_slug}}.py

```diff
@@ -26,30 +26,26 @@
 
 
 {% if cookiecutter.use_pytest == 'y' -%}
-class Test{{ cookiecutter.project_slug|title }}(object):
+@pytest.fixture
+def some_fixture():
+    pass
```

Comment:
We could have a bit more code here to showcase how this works, for users not already familiar with pytest fixtures?

Perhaps something like:

```
@pytest.fixture
def db_transaction():
    """Sample pytest fixture
    See more at: http://doc.pytest.org/en/latest/fixture.html
    """
    # setup code, e.g. db.beginTransaction()
    yield
    # teardown code, e.g. db.rollback()
```